documentation/docs/tutorials/setup.md

```diff
@@ -528,3 +528,142 @@ The Grafana dashboard should be accessible at `http://<your-ip>:3000`.
+
+```bash
+cd /opt
+git clone https://github.com/iotaledger/goshimmer.git
+```
+
+###  Golang-go 1.16
+
+Your system will need to have [Go 1.16](https://go.dev/doc/go1.16) installed for the build script to work. You can install it by running the following command:
+
+```bash
+apt install golang-go
+````
+
+You can make sure you are using the right go version by running the following command:
+
+```bash
+go version
+```
+
+If apt did not install Go version 1.16+, you can use the following commands to download and install it from source:
+
+
+```bash
+cd /home
+wget https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz
+sudo tar -xvf go1.16.4.linux-amd64.tar.gz
+sudo mv go /usr/local
+export GOROOT=/usr/local/go
+cd /opt/goshimmer
+mkdir gopath
+export GOPATH=/opt/goshimmer/gopath
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+You run `go version` again to check if you have successfully installed `golang-go 1.16`.
+This installation is only valid for this session. If you enter a new session, you will have to enter these commands again:
+
+```bash
+export GOROOT=/usr/local/go
+export GOPATH=/opt/goshimmer/gopath
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+
+### Download the Snapshot
+You can download the latest snapshot by running the following command from the goshimmer directory you created when you [cloned the repository](#clone-the-repository):
+
+
+```bash
+sudo wget -O snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/nectar/snapshot-latest.bin
+```
+
+### Making the Node Dashboard Accessible
+
+You will need to modify your goshimmer configuration file to make the Node Dashboard accessible. 
+1. Make a copy of the [config.default.json](https://github.com/iotaledger/goshimmer/blob/develop/config.default.json) file and rename it config.json.
+2. Using your text editor of choice, edit line 23 to change the `bindAddress` to `0.0.0.0:8081`.
+3. Save your changes.
+
+```bash
+nano config.default.json
+```
+
+On line `23` of the file, change the **bindAddress** to `0.0.0.0:8081`.
+
+Then press `CTRL+X` to exit.
+Then save the modified buffer (press `Y`).
+Then change the file name to `config.json` (you remove “default”).
+And press the `Y` key to save the file (this will allow you to view the GoShimmer dashboard in your browser).
+
+### Run the GoShimmer node
+
+You can now run the build script for the goshimmer binary with the following command:
+
+```bash
+./scripts/build.sh
+```
+
+:::tip 
+You can use the `screen` command to keep the node running if you terminate your current ssh session.
+:::
+
+
+
+You can now run the GoShimmer binary to start your node: 
+
+
+```bash
+./goshimmer
+```
+
+
+You can then "detach" from the GoShimmer screen by pressing your `CTRL+A+D` keys.  This will remove the GoShimmer window,  but it will still be running. 
```

Comment:
```suggestion
You can "detach" from the GoShimmer screen by pressing your `CTRL+A+D` keys. This will remove the GoShimmer window,  but it will still be running. 
```